tools/nni_cmd/nnictl_utils.py

```diff
@@ -509,10 +509,19 @@ def export_trials_data(args):
             # dframe = pd.DataFrame.from_records([parse_trial_data(t_data) for t_data in content])
             # dframe.to_csv(args.csv_path, sep='\t')
             records = parse_trial_data(content)
-            with open(args.csv_path, 'w') as f_csv:
-                writer = csv.DictWriter(f_csv, set.union(*[set(r.keys()) for r in records]))
-                writer.writeheader()
-                writer.writerows(records)
+            if args.type == 'json':
+                json_records = []
+                for trial in records:
+                    value = trial.pop('reward', None)
+                    del trial['id']
```

Comment:
no need to remove id, just keep it as the first column

Reply:
Done